src/main/java/iroha/validation/transactions/provider/impl/IrohaHelper.java

```diff
@@ -0,0 +1,92 @@
+package iroha.validation.transactions.provider.impl;
+
+import com.google.common.base.Strings;
+import io.reactivex.Observable;
+import iroha.protocol.BlockOuterClass;
+import iroha.protocol.QryResponses;
+import iroha.protocol.Queries;
+import iroha.protocol.TransactionOuterClass;
+import java.security.KeyPair;
+import java.time.Instant;
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import jp.co.soramitsu.iroha.java.BlocksQueryBuilder;
+import jp.co.soramitsu.iroha.java.IrohaAPI;
+import jp.co.soramitsu.iroha.java.Query;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+/**
+ * Class that helps with different iroha interaction
+ */
+@Component
+public class IrohaHelper {
+
+  private static final Logger logger = LoggerFactory.getLogger(BasicTransactionProvider.class);
```

Comment:
`BasicTransactionProvider.class` should not be here

Reply:
Ouch, soryan